decisions/log/0032-cross-sectoral-standards.md

```diff
@@ -0,0 +1,86 @@
+
+We have also considered the possibility of disclosing a cross sectoral standard not contained in the set of options explicitly covered by the Technical Specifications. We concluded that this possibility was suboptimal, for the following reasons:
+* It would afford more complexity to the Technical Specifications, requiring a structured data type similar to [`ProductOrSectorSpecificRule`](https://wbcsd.github.io/data-exchange-protocol/v2/#dt-productorsectorspecificrule)
+* There would be no clear guidance for data recipients who received a `ProductFootprint` with an unexpected cross sectoral standard
+* There are no straightforward advantages in this possibility that supersede the disadvantages of accommodating it
+
+
+## Detailed Technical Proposal
+
+1. Addition of deprecation warnings to the attribute `crossSectoralStandardsUsed`, as well as to the data types `CrossSectoralStandard` and `CrossSectoralStandardSet`.
+2. Addition of new property `crossSectoralStandards` defined as follows:
+
+    ```
+    <tr>
+        <td><dfn>crossSectoralStandards</dfn>
+        <td>Array of Strings
+        <td>M
+        <td>The cross-sectoral standards applied for calculating or allocating [=GHG=] emissions.
+
+        It MUST be a non-empty array and MUST contain only the following values without duplicates:
+
+        Recommended:
+        : ISO14067
+        :: for the ISO 14067 Standard, "Greenhouse gases — Carbon footprint of products — Requirements and guidelines for quantification"
+        : ISO14083
+        :: for the ISO 14083 Standard, "Greenhouse gases — Quantification and reporting of greenhouse gas emissions arising from transport chain operations"
+        : ISO14040-44
+        :: for the ISO 14040-44 Standard, "Environmental management — Life cycle assessment — Principles and framework"
+        : GHGP Product
+        :: for the Greehouse Gas Protocol (GHGP) Product Standard
+        : PEF
+        :: for the EU Product Environmental Footprint Guide
+        : PACT Framework v2
+        :: for version 2 of the PACT Framework
+        : PACT Framework v2.1
+        :: for version 2.1 of the PACT Framework
+        Not recommended but permitted*:
+        : PACT Framework v1
+        :: for version 1 of the PACT Framework
+        : PAS2050
+        :: for the Publicly Available Specification (PAS) 2050, "Specification for the assessment of the life cycle greenhouse gas emissions of goods and services"
+
+        Advisement:
+            The enumeration of standards above CAN evolve in future revisions. A host system MUST accept ProductFootprints from later revisions with `crossSectoralStandards` containing values that are not defined in this specification.
+            The set of standards listed include some standards which PACT does not recommend given they have become deprecated and/or outdated. However, with the goal of enabling transparency, PACT Technical Specifications nevertheless includes these as options.
```

Comment:
Is there a more structured way to indicated standards that are listed but not recommended due to deprecation? I think it would make sense to not have them all be viewed at a data level as the same, basically i think that the deprecation side of things should be codified

Reply:
Hmm yes that's the challenge, as some are formally deprecated (like Framework v1) whereas others are generally not recommended but not officially deprecated (PAS 2050). Do you have a suggestion on how to codify this differently (without adding too much complexity... :)